.travis.yml

```diff
@@ -89,6 +96,6 @@ script:
 
 after_script:
   - echo "after_script start"
-  - source activate pandas-dev && pushd /tmp && python -c "import pandas; pandas.show_versions();" && popd
-  - ci/print_skipped.py 
+  - source activate pandas-dev && pushd /tmp && if [ `uname -m` = 'aarch64' ];then VAR="python3.7" ; else VAR="python";fi && $VAR -c "import pandas; pandas.show_versions();" && popd
+  - ci/print_skipped.py
```

Comment:
can you undo the changes to the previous two files? I guess they were tests, but in any case, we don't want this.